homeassistant/components/amcrest/sensor.py

```diff
@@ -17,10 +19,15 @@
 
 SENSOR_PTZ_PRESET = "ptz_preset"
 SENSOR_SDCARD = "sdcard"
-# Sensor types are defined like: Name, units, icon
-SENSORS = {
-    SENSOR_PTZ_PRESET: ["PTZ Preset", None, "mdi:camera-iris"],
-    SENSOR_SDCARD: ["SD Used", PERCENTAGE, "mdi:sd"],
+
+
+SENSORS: dict[str, SensorEntityDescription] = {
+    SENSOR_PTZ_PRESET: SensorEntityDescription(
+        key="PTZ Preset", icon="mdi:camera-iris"
+    ),
+    SENSOR_SDCARD: SensorEntityDescription(
+        key="SD Used", unit_of_measurement=PERCENTAGE, icon="mdi:sd"
+    ),
 }
```

Comment:
```suggestion
SENSORS: tuple[SensorEntityDescription, ...] = (
    SensorEntityDescription(
        key=SENSOR_PTZ_PRESET,
        name="PTZ Preset",
        icon="mdi:camera-iris",
    ),
    SensorEntityDescription(
        key=SENSOR_SDCARD,
        name="SD Used",
        unit_of_measurement=PERCENTAGE,
        icon="mdi:sd",
    ),
)
```

Reply:
Done

Reply:
I did it, but the commit hooks, formats it back again, sorry.

Reply:
It doesn't. Also, the change here hasn't been applied, as it removes the dictionary. Unresolved this comment.

Reply:
Missed the last comma (I do not understand why the formatter is soo sensitive, but that is another story).

Changed.